src/game_of_life.js

```diff
@@ -0,0 +1,43 @@
+export function countLiveNeighbours(board, i, j) {
+  let count = 0;
+  for (let x = i - 1; x <= i + 1; x += 1) {
+    if (x < 0 || x >= board.length) continue; // eslint-disable-line no-continue
+    for (let y = j - 1; y <= j + 1; y += 1) {
+      if (y < 0 || y >= board[x].length) continue; // eslint-disable-line no-continue
+      count += board[x][y];
+    }
+  }
+
+  count -= board[i][j];
+  return count;
+}
+
+function next(input) {
+  const output = input.map((row) => row.map((cell) => cell));
+
+  for (let i = 0; i < input.length; i += 1) {
+    for (let j = 0; j < input[i].length; j += 1) {
+      const liveNeighbours = countLiveNeighbours(input, i, j);
+
+      if (liveNeighbours < 2) {
```

Comment:
I like the comments here because this is basically the core business logic.
Also I like the fact that you keep each rules as separate condition and so not try to 'optimise'  that as this is much more readable.